VirtoCommerce.Storefront/Domain/Cart/CartConverter.cs

```diff
@@ -761,10 +761,10 @@ public static cartDto.LineItem ToLineItemDto(this LineItem lineItem)
                 Currency = lineItem.Currency.Code,
                 Discounts = lineItem.Discounts.Select(ToCartDiscountDto).ToList(),
 
-                ListPrice = (double)lineItem.ListPrice.InternalAmount,
-                SalePrice = (double)lineItem.SalePrice.InternalAmount,
+                ListPrice = (double)lineItem.ListPrice.Amount,
```

Comment:
It is important to do not send to the API rounded money amount, why you change this to rounded?